src/appkit.cs

```diff
@@ -12913,6 +13297,39 @@ interface NSSliderCell {
 		[Export ("barRectFlipped:")]
 		CGRect BarRectFlipped (bool flipped);
 	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSSliderTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
```

Comment:
We had 	[DisableDefaultCtor] on the base NSTouchBarItem, but i'm adding it to all of the derived types.